dev-python/sphinx-autoapi/sphinx-autoapi-1.8.4.ebuild

```
@@ -0,0 +1,32 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..10} )
+
+inherit distutils-r1
+
+DESCRIPTION="A new approach to API documentation in Sphinx."
```

See proxy maintainers [style guide](https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Descriptive_text):
```suggestion
DESCRIPTION="A new approach to API documentation in Sphinx"
```